.eslintrc

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
I could ignore it but that could be confusing if we move to esint in the future.